### ISSUE TITLE
Demographics and Labels not translated

### ISSUE THREAD

**Issue by mboskin** [covid-alert-ny/covid-green-app#122]
[7:41 PM] Arshad, Fahd (DOH-IT-SWAT)
    None of the demographics are translated, and neither are some other labels like "Choose..." and "I am feeling well today.". Boskin, Michael (HEALTH) can you kindly enter a new issue for NF to make the demographic survey list items all translatable, and in parallel tell the NYS translators to kindly translate these these items? 
[COVID SMART NY_Translations_v1 final.xlsx](https://github.com/project-vagabond/covid-green-app/files/5139260/COVID.SMART.NY_Translations_v1.final.xlsx)
The list is not getting translated if I switch to Google-Japanese in the current build either.

**Comment by AlanSl** [comment]
Fixed in code, should be fixed in the next build (18?)

**Comment by kevin-sheldon** [comment]
Retested and all demographics (except for NYS County, which I think is fine) are now translated.

**Comment by mboskin** [comment]
Again, I appreciate you going through these. If you close it in GitHub would you mind closing it in Teams?


www.nystec.com<http://www.nystec.com/>

**Comment by mboskin** [comment]
Failed retesting as per following:

Patrick Brennan | 9/1/20 5:29 PM | Fail | On landing screen Percent tested positive still in original English as well as x and y axis titles. Data and Privacy Policy is also untranslated.

**Comment by AlanSl** [comment]
These aren't related to the original bug fixed above and most I believe have already been covered by recent translations updates:

- [x] Percent tested positive translates as expected for me on the latest build
- [x] Other chart labels translate as expected 
- [x] Abridged calendar month names currently don't translate, <strike>I'll fix that</strike> https://github.com/project-vagabond/covid-green-app/pull/323
- [x] In-app short form privacy policy does translate as expected

**Comment by AlanSl** [comment]
@mboskin The chart x-axis label dates (and also exposure alert date and checker history dates) should now be correctly translated in the next build.

This will show for all languages except Haitian Creole for which date format translations aren't readily available (those fall back to English for now).

**Comment by mboskin** [comment]
Raghunath Gajjala | Pass | Tested. Issue resolved.
